js_modules/dagster-ui/packages/ui-core/src/assets/AssetsCatalogTable.tsx

```diff
@@ -122,6 +122,8 @@ type FilterType = {
   tags: DefinitionTag[];
 };
 
+const emptyArray: any[] = [];
```

Comment:
It would be nice for this to be EmptyArray or EMPTY_ARRAY to indicate that it's a file constant and not a local var